1-mandatory/2-quotegenerator/index.html

```diff
@@ -12,6 +12,23 @@
   </head>
   <body>
     <!-- Write your HTML in here -->
+   <section>
+    <div>
+      <div class="container">
+        <h2 id="quote">"Life isn't about getting and having, it's about giving and being."</h2>
+        <p id="author">- Kevin Kruse</p>
+        <button id="next" type="button">Next</button>
+        
+        <div class="wrap">
+          <p id="autoPlay">Auto-play: OFF</p>
+          <label class="switch">
+            <input id="toggle" type="checkbox">
+            <span class="slider round"></span>
+          </label>
+        </div>
+      </div>
+    </div>
+   </section>
```

Comment:
Comment: You also use a class container for your html. I wonder can you not just remove the section and div above on line 15 and 16. I dont know what purpose they fill. Then you could just start with section class="container". Should work just fine without. Well done!